api/src/services/nfl-teams.service.ts

```diff
@@ -0,0 +1,22 @@
+import { hash } from 'bcrypt';
+import { Service } from 'typedi';
+import { DB } from '@database';
+import { CreateNFLTeamDto } from '@dtos/nfl-teams.dto';
+import { HttpException } from '@/exceptions/httpException';
+import { NFLTeam } from '@interfaces/nfl-teams.interface';
+
+@Service()
+export class NFLTeamService {
+  public async findAllNFLTeam(): Promise<NFLTeam[]> {
+    const allNFLTeam: NFLTeam[] = await DB.NFLTeams.findAll();
```

Comment:
How do the findAll, and findByPK work? Are these sequalize functions.

Reply:
Yes, the `findAll` and `findByPk` are Sequelize functions. When they are called, they generate a `SELECT` SQL query and run it to grab the information form the database. Here is more information on it: https://sequelize.org/docs/v6/core-concepts/model-querying-finders/#findone